cmd/build.go

```diff
@@ -54,8 +54,12 @@ func newLocalExecuteCommand(config *settings.Config) *cobra.Command {
 	}
 
 	buildCommand := &cobra.Command{
-		Use:   "execute",
+		Use:   "execute [flags] [-- Docker specific flags]",
```

Comment:
I don't think calling these "docker specific" is correct. It should be the flags that are specified on this command.

Reply:
But they are. Any flags here would be what's available with you run `docker run`. They are passed to the Docker CLI.